src/services/user_api/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "ft-user-api-setup",
+  "version": "0.0.1",
+  "description": "Api that allows users to login, create acc, delete account,authenticate account",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "farmrail",
+  "license": "MIT",
+  "dependencies": {
+    "dotenv": "^8.2.0",
+    "express": "^4.17.1",
+    "mongoose": "^5.9.12"
+  },
+  "devDependencies": {
```

Comment:
add @babel to your dependencies so as you can easily use ES6 functionality in your code. Also, learn more about **@babel build, @babel/node, @babel/preset-env**

Reply:
> add @babel to your dependencies so as you can easily use ES6 functionality in your code. Also, learn more about **@babel build, @babel/node, @babel/preset-env**

Duly Noted 👍  added!